packages/docusaurus-theme-classic/src/theme/NavbarItem/DocNavbarItem.tsx

```diff
@@ -45,14 +45,19 @@ export default function DocNavbarItem({
     [activeVersion, preferredVersion, latestVersion].filter(Boolean),
   );
   const doc = getDocInVersions(versions, docId);
+  const isCurrentDoc = activeDoc && activeDoc.sidebar === doc.sidebar;
+  const activeDocInfimaClassName = props.mobile
+    ? 'menu__link--active'
+    : 'navbar__link--active';
+  props.activeClassName = activeDocInfimaClassName;
```

Comment:
in this case, it's harmless but it's a bad practice to mutate React props. It would be better to create an extra variable